EXIOBASE_conversion_software/csv2rdf.py

```diff
@@ -32,31 +32,56 @@
 import numpy as np
 import pandas as pd
 
+from os import listdir
+from os.path import isfile, join
+import time
+
 from rdflib import Graph, Literal, BNode, Namespace, URIRef
 from rdflib.namespace import DCTERMS, FOAF, XSD, OWL, RDFS, RDF, SKOS
 
+def merge_files(args, filename):
+    outdir = args.outdir
+    if not outdir.endswith('/'):
+        outdir = outdir + '/'
+
+    ntFiles = [f for f in listdir(outdir) if isfile(join(outdir, f)) and ".nt" in f]
+
+    f = open("{}{}_merged.nt".format(outdir, filename), "w")
+    for file in ntFiles:
+        print("Working on file: {}".format(file))
+        start = time.time()
+        with open("{}{}".format(outdir, file), "r") as fp:
+            line = fp.readline()
+            while line and line.strip() != "":
+                f.write(line)
+                line = fp.readline()
+        end = time.time()
+        print("Time elasped", end - start)
+
+    print("All graphs merged")
+
+
 def file_name(path):
     head, tail = ntpath.split(path)
     return tail or ntpath.basename(head)
 
 
-def makeRDF(data, code="HSUP", isInput=True):
-    '''
-    create an RDF graph from a Pandas dataframe of 11 columns.
-    the dataframe is output of the `xslb2csv` function.
-    the code specifies which file is this taken from HSUP/HUSE/Other?
-    isInput=true -> the data is about input/use flows
-    isInput=false -> the data is about output/supply flows
+def serialize_graph(args, filename, graph, index=None):
+    outdir = args.outdir
+    if not outdir.endswith('/'):
+        outdir = outdir + '/'
 
-    # Data row:
```

Comment:
Has this documentation about the file structure been kept somewhere?

Reply:
@kuzeko Yes, in the `makeRDF` method declaration.